product_docs/docs/pgd/5/reference/routing.mdx

```diff
@@ -38,14 +38,14 @@ bdr.alter_proxy_option(proxy_name text, config_key text, config_value text);
 -   `config_key` &mdash; Key of the option in the proxy to be changed.
 -   `config_value` &mdash; New value to be set for the given key.
 
-The proxy options that can be changed using this function are:
-- `listen_address` &mdash; Address for the proxy to listen on. Default is '{0.0.0.0}'.
-- `listen_port` &mdash; Port for the proxy to listen on. Default is '6432'.
-- `max_client_conn` &mdash; Maximum number of connections for the proxy to accept. Default is '32767'.
-- `max_server_conn` &mdash; Maximum number of connections the proxy can make to the Postgres node. Default is '32767'.
-- `server_conn_timeout` &mdash; Connection timeout for server connections. Default is '2' (seconds).
-- `server_conn_keepalive` &mdash; Keepalive interval for server connections. Default is '10' (seconds).
-- `consensus_grace_period` &mdash; Duration for which proxy continues to route even upon loss of a Raft leader. If set to 0s, proxy stops routing immediately. Default is generally '6' (seconds) for local proxies and '12' (seconds) for global proxies. These values will be overriden if `raft_response_timeout`, `raft_global_election_timeout` or `raft_group_election_timeout` are changed from their defaults.
+The proxy options that you can change using this function are:
+- `listen_address` &mdash; Address for the proxy to listen on. Default is `{0.0.0.0}`.
+- `listen_port` &mdash; Port for the proxy to listen on. Default is `6432`.
+- `max_client_conn` &mdash; Maximum number of connections for the proxy to accept. Default is `32767`.
+- `max_server_conn` &mdash; Maximum number of connections the proxy can make to the Postgres node. Default is `32767`.
+- `server_conn_timeout` &mdash; Connection timeout for server connections, in seconds. Default is `2`.
+- `server_conn_keepalive` &mdash; Keepalive interval for server connections, in seconds. Default is `10`.
+- `consensus_grace_period` &mdash; Duration for which proxy continues to route even upon loss of a Raft leader, in seconds. If set to 0s, proxy stops routing immediately. Default is generally `6` for local proxies and `12` for global proxies. These values are overridden if `raft_response_timeout`, `raft_global_election_timeout`, or `raft_group_election_timeout` are set to values other than their defaults.
```

Comment:
The "in seconds" edit...is that what was meant? It seemed more apt to qualify it in the description rather than in the defaults, if that's correct.